.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore the default AAS storage directory
+storage/
+
+# Ignore PyCharm IDE configuration
+.idea
```

Comment:
Can you add an empty line at the end of the file? 

Fun fact: Actually the new line at the end of files is not demanded by programming style, but stems from the [POSIX definition of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206). Therefore this rule applies to all text files. Apparently, some older unix tools and some git diff tools have problems if that new line character is missing.

Reply:
Ahh, thank you for explaining! I did not know about that at all.